### PR TITLE
Add Rich traceback

### LINKED ISSUE #552: [BUG-General] Terminal output not formatted and uses ASCII in windows
**Describe the bug**
I am on Windows and command prompt and I `cd` into example_scenes directory and run 
```sh
manim basic.py
``` 
and I get a output like below.
teminal-format
I should get in green colour though.

**To Reproduce**
Just running the one in example_scene in enough.

**Expected behavior**
The ill formatted thing should be in green colour.

**Logs**
<details><summary>Terminal output (Screenshots acceptable)</summary>

teminal-format

</details>

**System Specifications**

<details><summary>System Details</summary>

- OS (with version, e.g Windows 10 v2004 or macOS 10.15 (Catalina)): Windows 7
- Python version (`python/py/python3 --version`): 3.8

### PR DESCRIPTION
**Pull request by naveen521kk**
## List of Changes
Rich tracebacks are super cool and this is a setup to implement it.
image
Also, fixed #552

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

### CODE REVIEW

File: manim/config/logger.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def set_file_logger(log_file_path):
 logger = logging.getLogger("manim")
 # The console is set to None as it will be changed by set_rich_logger.
 console = None
-
+install()
```
Comment (leotrs) [inline manim/config/logger.py:117]:
```suggestion
# Install the rich traceback subpackage. This is necessary because **reasons here**.
install()
```

Comment (leotrs) [reply manim/config/logger.py:117]: Weird that it requires an "install" but whatever I guess. Does this "install" take long?

Comment (naveen521kk) [reply manim/config/logger.py:117]: > Does this "install" take long?

I don't think so.

But what it does is add *beautiful* tracebacks. See screenshots.

**Comment by leotrs** [comment]
The only problem I have with this is that it is essentially non-testable. But the positives do seem to outweigh the negatives.

**Review by leotrs — APPROVED** [review]

**Review by eulertour — APPROVED** [review]

**Review by kilacoda — APPROVED** [review]

**Pull request merged**